confgenerator/testdata/invalid/linux/logging-pipeline_reserved_id_prefix/golden_error

```diff
@@ -1 +1,7 @@
-logging pipeline id "lib:pipeline_1" is not allowed because prefix 'lib:' is reserved for pre-defined pipelines.
\ No newline at end of file
+the agent config file is not valid. detailed error: [10:21] Key: 'LoggingService.Pipelines[lib:pipeline_1]' Error:Field validation for 'Pipelines[lib:pipeline_1]' failed on the 'startsnotwith' tag
```

Comment:
Are we tracking the error messages to fix later?

Reply:
"all of them" :)